src/main/resources/assets/pyrotech/lang/zh_cn.lang

```diff
@@ -350,16 +352,18 @@ gui.pyrotech.tooltip.torch.burns.false=持续燃烧。
 gui.pyrotech.tooltip.rain.extinguishes.true=淋雨后熄灭，
 gui.pyrotech.tooltip.rain.extinguishes.false=淋雨后仍燃烧。
 gui.pyrotech.tooltip.potency=Potency: %d%%
-gui.pyrotech.tooltip.redstone.activation.chance=Has a %s%d%%%s chance to activate when damaged.
-gui.pyrotech.tooltip.redstone.active.durability=Has a %s%d%%%s chance to ignore damage while active.
-gui.pyrotech.tooltip.redstone.active.efficiency=Breaks blocks %s%d%%%s faster while active.
-gui.pyrotech.tooltip.redstone.active.damage=Deals %s%d%%%s damage while active.
-gui.pyrotech.tooltip.redstone.active.hoe=Tills a %s3x3%s area while active.
-gui.pyrotech.tooltip.compost.value=Compost Value: %d
-
-gui.pyrotech.tooltip.quartz.active.efficiency=Breaks blocks %s%d%%%s faster in %sThe Nether%s.
-gui.pyrotech.tooltip.quartz.active.damage=Deals %s%d%%%s damage in %sThe Nether%s.
-gui.pyrotech.tooltip.quartz.active.hoe=Tills a %s3x3%s area in %sThe Nether%s.
+#I (DRAMING)couldn't find this entry, so I didn't translate it.(Could you please tell me?)
```

Comment:
The potency tooltip describes the magnitude of the effect that eating a Roasted Marshmallow has.

Reply:
But how can I roast a marshmallow? I couldn't find any recipes in the Jei.(I do not really play Pyrotech well, lol)

Reply:
Craft a stick with a sword or axe to get a sharpened stick. Place sharpened stick in main hand and marshmallow in offhand and click to combine. Stand near campfire and use combined stick / marshmallow on the campfire. Be careful not to burn it.

Reply:
Okay. I see. Thanks a lot!!